.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,64 @@
+name: Bug Report
+description: Found a Bug that needs to be fixed?
+body:
+- type: markdown
+  attributes:
+    value: |-
+      Please join the [Discord Server](https://discord.gg/0hMr4ce0tIl3SLv5) for questions or ask them in [our Discussions](https://github.com/DV8FromTheWorld/JDA/discussions).
+      
+      Keep in mind that this isn't the place to learn Java.  
+      Please head over to [StackOverflow](https://stackoverflow.com/questions/tagged/java) for your general programming questions.
+- type: checkboxes
+  attributes:
+    label: General Troubleshooting
+    description: You confirm to have made the following checks first.
+    options:
+      - label: I have checked for similar issues on the Issue-tracker.
+        required: true
+      - label: I have updated to the [latest JDA version](https://bintray.com/dv8fromtheworld/maven/JDA/_latestVersion) # TODO: Update to new URL once changed.
```

Comment:
We no longer use bintray

Reply:
Should I update this to the Jenkins then? Or better the mavenCentral location?

Reply:
Probably jenkins since it has the latest fixes

Reply:
Changed to Jenkins URL of latest successful build